requirements/requirements.txt

```diff
@@ -1,8 +1,9 @@
 pillow==8.3.2
 nncf>=2.0.0
 pytorch-lightning==1.3.6
-torch==1.8.1
-torchvision==0.9.1
+-f https://download.pytorch.org/whl/torch_stable.html
+torch==1.10.0+cu111
+torchvision==0.11.0+cu111
```

Comment:
As per our discussion, I guess we'll remove cu111. We might need to check with ote that they are fine with these version.